### PR TITLE
Move balancer tcl to vault_msig & stake dust

### PR DESCRIPTION
**Pull request by petrovska-petro**
Tackles #488 

```
brownie run issue/488/transfer_balancer_tcl
```

### CODE REVIEW

File: scripts/issue/488/transfer_balancer_tcl.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    )
+    vault.take_snapshot(tokens=[bpt_20BTC_80_BADGER_GAUGE.address])
+
+    trops.balancer.stake_all(bpt_20BTC_80_BADGER)
+
+    bal = bpt_20BTC_80_BADGER_GAUGE.balanceOf(trops)
+
+    bpt_20BTC_80_BADGER_GAUGE.transfer(vault, bal)
+
+    vault.print_snapshot()
```
Comment (jorijnsmit) [inline scripts/issue/488/transfer_balancer_tcl.py:26]:
```suggestion
    trops.print_snapshot()
    vault.print_snapshot()
```

Comment (petrovska-petro) [reply scripts/issue/488/transfer_balancer_tcl.py:26]: this is not strictly needed as the snapshot was taken lines above and will contain the `snapshot` attribute and this line will get exec: https://github.com/Badger-Finance/badger-multisig/blob/main/great_ape_safe/great_ape_safe.py#L248

Comment (jorijnsmit) [reply scripts/issue/488/transfer_balancer_tcl.py:26]: ah sure. just a bit annoying to not have them next to each other to compare.

**Review by jorijnsmit — APPROVED** [review]

**Review by sajanrajdev — APPROVED** [review]
- Stakes dusty balance of pool token on Balancer from trops
- transfers full amount of staked position from TechOps to treasury vault

LGTM

**Pull request merged**